modules/_labs/combobox/react/stories/stories.tsx

```diff
@@ -44,8 +44,9 @@ class Autocomplete extends React.Component<
         labelId="autocomplete-123"
         onFocus={action(`Focus`)}
         onBlur={action(`Blur`)}
+        initialValue={`Test`}
```

Comment:
would this work? initialValue='Test'

Reply:
moved to single quotes more consistently